packages/frontend/src/router/index.ts

```diff
@@ -18,6 +18,16 @@ const routes = [
     name: 'new-room',
     component: () => import(/* webpackChunkName: "new-room" */ '../views/NewRoom.vue'),
   },
+  {
+    path: '/book-desk',
+    name: 'bookDesk',
+    component: () => import(/* webpackChunkName: "BookDesk" */ '../views/BookDesk.vue'),
+  },
+  {
+    path: '/book-desk/modify',
+    name: 'modify',
+    component: () => import(/* webpackChunkName: "modify" */ '../views/Modify.vue'),
```

Comment:
I would remove the word desk from that component / route completely, so we could use this for booking other things as desks as well.

```suggestion
    path: '/booking/edit',
    name: 'booking-edit',
    component: () => import(/* webpackChunkName: "booking-edit" */ '../views/BookingEdit.vue'),
```